htdocs/core/lib/pdf.lib.php

```diff
@@ -2284,7 +2284,17 @@ function pdf_getLinkedObjects($object, $outputlangs)
 			}
 		} elseif ($objecttype == 'commande' || $objecttype == 'supplier_order') {
 			$outputlangs->load('orders');
-			foreach ($objects as $elementobject) {
+
+			if (count($objects) > 1) {
+				$object->note_public .= '<br/>'.$outputlangs->transnoentities("RefOrder").' : <br/>';
```

Comment:
You must use <br> that is html5 tag and not <br/> that is xml tag (even if <br/> is accepted by browsers).
Also, you must use dol_concatdesc to add text to a note_public (because not can be a full text or html text and concatenation need different action when you add a html text on a non html text. dol_concatdesc do it for you.

Reply:
done